index.html

```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
+            </div>
+            <div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="prog-lang">
+                <li class="lang">html</li>
+                <li class="lang">css</li>
+                <li class="lang">javascript</li>
+            </ul>
+            <button class="project-link" type="button">See Project</button>
+        </div>
+        <div class="card-works">
+            <img class="snapshot" src="work-snapshot-2.png" alt="project-snapshot">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="caption-container">
+                <div class="text">CANOPY</div>
+                <div class="dot">&middot;</div>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
+            </div>
+            <div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="prog-lang">
+                <li class="lang">html</li>
+                <li class="lang">css</li>
+                <li class="lang">javascript</li>
+            </ul>
+            <button class="project-link" type="button">See Project</button>
+        </div>
+        <div class="card-works">
+            <img class="snapshot" src="work-snapshot-3.png" alt="project-snapshot">
+            <h2 class="project-title">Tonic</h2>
+            <div class="caption-container">
+                <div class="text">CANOPY</div>
+                <div class="dot">&middot;</div>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
+            </div>
+            <div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="prog-lang">
+                <li class="lang">html</li>
+                <li class="lang">css</li>
+                <li class="lang">javascript</li>
+            </ul>
+            <button class="project-link" type="button">See Project</button>
+        </div>
+        <div class="card-works">
+            <img class="snapshot" src="work-snapshot-4.png" alt="project-snapshot">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="caption-container">
+                <div class="text">CANOPY</div>
+                <div class="dot">&middot;</div>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
```

Comment:
- [ ] You should stick to the design as much as possible like font, color, spacing, etc. See the color of `Back End Dev and 2015`. See image below for reference
### Your design

![your](https://user-images.githubusercontent.com/81727988/212335025-08fb414a-c193-46fc-b5de-7d79f0e6cc19.jpg)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/212335055-e6dac61c-e1c0-4c5f-ada2-3c3656d7b6ca.jpg)

Reply:
![canopy](https://user-images.githubusercontent.com/47403388/212368868-4709f9ab-041c-403a-b841-05c8251d39c1.jpg)